feature-libs/customer-ticketing/assets/translations/en/customer-ticketing.ts

```diff
@@ -11,16 +11,19 @@ export const customerTicketing = {
     message: 'Message',
     cancel: 'Cancel',
     submit: 'Submit',
-    charactersLeft: 'Characters Left: {{count}}',
+    charactersLeft: 'characters left: {{count}}',
```

Comment:
We should separate this into another object specifying that this messages are for ticket creation

Example:
```
createCustomerTicket: {
 charactersLeft: 'characters left: {{count}},
 }
```

Reply:
Done